README.md

```diff
@@ -10,6 +10,76 @@ It can be used by installing this package & setting the `extends` property in yo
 }
 ```
 
+Alternative yml config  `.eslintrc.yml`:
```

Comment:
We've decided to use YML ESlint config format in Gambit and Blink because it supports comments, as opposed to JSON. The reasoning is "if you need to deviate from recommended standart, you should explain why".